.envrc

```diff
@@ -20,7 +20,6 @@ dotenv_if_exists .env
 if [ ! -d venv ]; then
     python3 -m venv venv
     venv/bin/pip install wheel
-    venv/bin/pip install -r requirements.txt
     venv/bin/pip install -r requirements.dev.txt
```

Comment:
In my previous commit, I had renamed this file to be `requirements.dev-prod.txt`, however, Github not understand the move. I will follow up.